CITATION.cff

```diff
@@ -0,0 +1,66 @@
+cff-version: 1.2.0
+message: "If you use this software in your work, please cite it using the following metadata."
+authors:
+- family-names: "Graham"
+  given-names: "Brett"
+  orcid: "https://orcid.org/0000-0001-6315-4507"
+- family-names: "Burnett"
+  given-names: "Zachary"
```

Comment:
```suggestion
- family-names: "Burnett"
  given-names: "Zachary"
  orcid: "https://orcid.org/0000-0002-1854-5813"
```

Reply:
Since we're autogenerating the contributors with the zenodo integration you'll need to link your orcid with your github name for this association to be made. Let me know if I can help.